format/view-spec.md

```diff
@@ -118,6 +118,31 @@ This type of representation stores the original view definition in SQL and its S
 For `CREATE VIEW v (alias_name COMMENT 'docs', alias_name2, ...) AS SELECT col1, col2, ...`,
 the field aliases are 'alias_name', 'alias_name2', and etc., and the field docs are 'docs', null, and etc.
 
+#### Atomic swap for File System Views
```

Comment:
why do we want to distinguish between file system views and metastore veiws? To me those are just different implementations of the view catalog, and how they ensure the operation is atomic is an implementation detail of the specific view catalogs.

Reply:
From what I understand it is the same as for tables, which have [File system tables](https://iceberg.apache.org/spec/#file-system-tables) and [Metastore tables](https://iceberg.apache.org/spec/#metastore-tables). I just included it for completeness. Since the the ecosystem is moving towards metastore tables and views we can also leave this section out.

Reply:
I think that is written under the context of commit conflict resolution, and those are specific examples about how the commit resolution could be implemented as atomic swap in file system or metastore. But with introduction of REST catalogs that are backed by complicated services, there could be much more complex operations done during commit time. This makes me think if we should also update the table spec to make this clear that these are more like examples, but not specifications.